manifests/bases/image-automation-controller/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
You likely want to automate updates for this in future releases, this can be done by adding a line to:

https://github.com/fluxcd/flux2/blob/main/.github/workflows/update.yml#L42

Reply:
In a different way to https://github.com/fluxcd/flux2/pull/580/commits/9d93adae72c0670b91a1c294f8ef9d6ca9f4df40#diff-571e661e1640e6f1f415c087c9a72af346ea9fd4655ff5545e5ad6ea596bab97R43 ?

Reply:
:100: